src/Indicator.vala

```diff
@@ -330,6 +335,45 @@ public class Session.Indicator : Wingpanel.Indicator {
         current_dialog.set_transient_for (indicator_icon.get_toplevel () as Gtk.Window);
         current_dialog.show_all ();
     }
+
+    private async void update_tooltip () {
+        string description;
+
+        if (server_type == Wingpanel.IndicatorManager.ServerType.SESSION) {
+            if (active_user_real_name == null) {
+                debug ("active_user is null. Getting active user");
+                active_user_real_name = Environment.get_real_name ();
+            }
+
+            int n_online_users = (yield manager.get_n_active_and_online_users ()) - 1;
+
+            // If active user is still null, fallback to "Unknown"
+            if (active_user_real_name == null) {
+                warning ("Active user null. Cannot get real name");
+                active_user_real_name = _("Unknown");
+            }
```

Comment:
```suggestion
            if (active_user_real_name == null) {
                active_user_real_name = Environment.get_real_name ();
            }

            int n_online_users = (yield manager.get_n_active_and_online_users ()) - 1;
```

According to [Valadoc](https://valadoc.org/glib-2.0/GLib.Environment.get_real_name.html):

- `GLib.Environment.get_real_name ()` will never return null
- `GLib.Environment.get_real_name ()` will return `Unknown` if the real user name can't be determined

So I wonder if we really need double null-check and fallback here.

Also let's remove (maybe) unnecessary debug message.

Reply:
Would there be any concerns regarding translation? i.e. would the string "Unknown" be translated if the real name cannot be determined?

Reply:
As far as seeing [the source code of glib](https://gitlab.gnome.org/GNOME/glib/-/blob/master/glib/gutils.c#L723) and its translation files, the string would not be returned as translated.